classifiers/reference_quality/chunked_sentence_complexity/code_snippet_common.md

```diff
@@ -0,0 +1,85 @@
+```python
+from collections import Counter
+import textstat
+import spacy
+
+def sentence_complexity(text: str)-> str:    
+    """
+    @param text: text to check
+    @return: either 'very difficult', 'difficult', 'fairly difficult', 'standard', 'fairly easy', 'easy' or 'very easy' depending on the score
+    """
+    score = textstat.flesch_reading_ease(text)
+    if score < 30:
+        return "very difficult"
+    if score < 50:
+        return "difficult"
+    if score < 60:
+        return "fairly difficult"
+    if score < 70:
+        return "standard"
+    if score < 80:
+        return "fairly easy"
+    if score < 90:
+        return "easy"        
+    return "very easy"
+
+def chunked_sentence_complexity(text: str, language: str = "en", spacy_model: str = "en_core_web_sm") -> str:
+    """
+    @param text: 
+    @param language: iso language code
+    @spacy model: name of a language model from SpaCy 
+    @return: aggregated reading ease score of a whole text
+    """
+    textstat.set_lang(language)
+    nlp = spacy.load(spacy_model)
```

Comment:
constantly reloading will results in overhead. A singleton like structure (see suggestion code) would reduce overhead

Reply:
Included the singleton like function in the common code.